packages/components/src/FormField/FormField.css

```diff
@@ -207,20 +212,22 @@
 }
 
 .textarea .label {
-  top: var(--field--padding-top);
+  width: calc(100% - var(--field--padding-right));
```

Comment:
Also, should remove the padding right on this label as well since the label will get cut off pretty early on when there's at least 16px of useable space on it.
![May-03-2023 16-02-55](https://user-images.githubusercontent.com/15986172/236059792-0d652997-a256-42d8-bbde-bef2b894d9bb.gif)

Reply:
Good call on the padding. I will add a comment

Reply:
Creates a tight edge on the right side when alignment is set to right, but I have no idea why we support alignment tbh, so going to ignore that.

![image](https://user-images.githubusercontent.com/10064579/236240773-a011fb55-40cc-468f-9c4d-391399081e8c.png)

Reply:
Will also create this gap on large fields when the field padding changes because it's larger than the scrollbar

![image](https://user-images.githubusercontent.com/10064579/236242249-03e1adf0-a97a-4fea-8c0a-be8099693c30.png)